product.go

```diff
@@ -6,7 +6,7 @@ import (
 )
 
 const productsBasePath = "admin/products"
-const resourceName = "products"
+const productsResourceName = "products"
```

Comment:
is this only for clarify? any plan to send in a PR to go-shopify?

Reply:
The resource name is used to construct the endpoint URL (`/admin/{resource_name}/{id}/metafields.json`). It's already in getconversio: https://github.com/getconversio/go-shopify/blob/master/util.go#L38

Reply:
I had to rename it because I'm using it in order as well. Can't have the same name because it's in the same package. So there's `productsResourceName` and `ordersResourceName`.